src/api/block_builder/input_selection/sender_issuer.rs

```diff
@@ -81,12 +44,12 @@ impl<'a> ClientBlockBuilder<'a> {
                         self.coin_type,
                         self.account_index,
                         self.input_range.clone(),
-                        &Address::Ed25519(*address),
+                        &Address::Ed25519(address),
```

Comment:
You don't have this copy if you rename `address` to `ed25519_address` because then you can use the original address